### ISSUE TITLE
rt upload always returns exit code 0

### ISSUE THREAD

**Issue by dquist** [jfrog/jfrog-cli#470]
Hi, it appears that the `rt upload` command always returns an exit code 0 even when the command reports an error such as a missing file or invalid destination.

CLI Version: 1.29.1

### Reproduce

```
jfrog rt u invalid_file invalid_dest
```

### What happens?

I get the following output indicating an error, but the exit code is 0.

```json
{
  "status": "failure",
  "totals": {
    "success": 0,
    "failure": 0
  }
}
```

## What should happen

The terminal should return an exit code 1 to indicate that an error occurred. 

This is fairly important since our build script will continue processing even if the upload has failed.

**Comment by Postyy** [comment]
Fix for this issue: https://github.com/jfrog/jfrog-cli/pull/474

**Comment by dquist** [comment]
Thanks for the quick fix! Should this issue be closed now?

**Comment by eyalbe4** [comment]
Sure. The fix is included in version **1.30.0**.
Thanks for fixing this @Postyy and thanks for reporting it @dquist!
